README.md

```diff
@@ -1,239 +1,99 @@
-# Secure Credential Store Plug-in for Zowe CLI
+# Kubernetes Secrets for Zowe CLI & Zowe Explorer
 
-<img src="imgs/Zowe_ConformanceBadge_cli.png" width=25% alt="Zowe Conformance Badge"/>
+The Kubernetes Secrets plugin for Zowe CLI and Zowe Explorer allows you to store your Zowe profile credetials as Kubernetes secrets in your Kubernetes cluster of choice. This is by changing the default credential manager upon installation into Zowe CLI or Zowe Explorer.
 
-**Note:** This plugin is not supported for Zowe v2 LTS because the secure credential manager is now built in to Zowe CLI. As long as Zowe v1 LTS is supported, it is still maintained and will receive bug fixes and security patches.
+This plugin can also be used as a sample to build your own custom credential manager to use based on your own use case.
 
-The Secure Credential Store Plug-in for Zowe CLI lets you store your credentials securely in the default credential manager in your computer's operating system.
+## How the plugin works
 
-As a systems programmer or application developer, you can store your credentials securely to help prevent your user name and password from being compromised as a result of a malware attack or unlawful actions by others.
+The plugin uses the Kubernetes API in order to communicate with your Kubernetes cluster and store secrets on a per user basis.
 
--   [How the plug-in works](#how-the-plug-in-works)
--   [Software requirements](#software-requirements)
--   [Installing](#installing)
--   [Building from source](#building-from-source)
--   [Securing Zowe CLI profiles](#securing-zowe-cli-profiles)
--   [Running tests](#running-tests)
--   [Uninstalling](#uninstalling)
--   [Contributing](#contributing)
+In addition, when storing Kubernetes secrets, the secret will be stored in the assigned namespace for the current context or otherwise through an environment variable named `$DEVWORKSPACE_NAMESPACE` and `$WORKSPACE_NAMESPACE`.
 
+## Requirements
 
-## How the plug-in works
+Before you install and use the plug-in make sure you have the following installed:
```

Comment:
please define what needs to be installed -- is it "applications"? is it "items"? Rewrite to say: Make sure you have the following applications/items/whatever installed:"